tests/integration/standard/test_shard_aware.py

```diff
@@ -0,0 +1,221 @@
+# Copyright DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import time
+import random
+from subprocess import run
+
+try:
+    from concurrent.futures import ThreadPoolExecutor, as_completed
+except ImportError:
+    from futures import ThreadPoolExecutor, as_completed  # noqa
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest  # noqa
+
+from cassandra.cluster import Cluster
+from cassandra.policies import TokenAwarePolicy, RoundRobinPolicy, ConstantReconnectionPolicy
+from cassandra import OperationTimedOut
+
+from tests.integration import use_cluster, get_node
+
+
+def setup_module():
+    os.environ['SCYLLA_EXT_OPTS'] = "--smp 4 --memory 2048M"
+    use_cluster('shared_aware', [3, ], start=True)
```

Comment:
Command not needed in a list literal, i.e. `[3]` is same as `[3,]`.
For comparison, it would be needed in a tuple, `(3)` is not same as `(3, )`

Reply:
Yeah, I was playing with multi cluster setup and probably forgot that